ParabolInc/parabol
# GitLab Oauth2 Poker Integration

Fix #5404 


#### To Test

- copy the GitLab env vars from the devops repo under `environments/local` and paste them into your `env` file in the `parabol` repo. If you don't have access to the devops repo, send a message in Slack requesting access
- if you're a super user, go to /admin/graphql and execute the `addFeatureFlag` mutation, adding gitlab. You can add yourself as a super user in postgres `localhost:5050`
- you may also need access to Parabol's GitLab app. I've invited Georg with have owner privileges and sent an email to elias@parabol.co (if that's the right email?) with guest privileges
- return to the `parabol` repo and run `yarn postdeploy`. This will run the `postDeploy` script which will call `primeIntegrations.ts` which will add the GitLab Parabol app credentials to your local db
- create a Sprint Poker meeting, go to the `Scope` phase, click on the `GitLab` tab, and click `Import issues from GitLab`. This should open the GitLab oauth2 page and, once you've clicked `Authorize`, you should see `Integrated!` within the GitLab card
